bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissNetworkParameter.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.souliss.internal.network.typicals;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.DatagramSocket;
+import java.util.Properties;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class contain parameter of Souliss Network. Those are loaded at startuo
```

Comment:
can you please explain how this class is intended to be used? Where is it filled?